SquareRegisterSDK/SCCAPIResponse.h

```diff
@@ -68,11 +68,24 @@
 @property (nonatomic, copy, readonly, nullable) NSString *offlinePaymentID DEPRECATED_MSG_ATTRIBUTE("Developers should pass the contents of the transactionID field in a Connect V2 request to retrieve the entire transaction.  This field will be removed in a later version.");
 
 /**
- The ID of the transaction.
+ The ID of the transaction, generated by Square's servers.  
+ Use this value to perform a Connect V2 request to retrieve the details of the transaction.
  */
 @property (nonatomic, copy, readonly, nullable) NSString *transactionID;
 
 /**
+ The client ID of the transaction, generated by the Square Register app.  
+ This value is generated for bookkeeping purposes, in case the transaction cannot immediately be completed.
+
+ If the transaction was done in offline mode or asynchronously (e.g., for cash or other tender transactions),
+ the server-generated <code>transactionID</code> may not be available immediately.  Developers can use this value
```

Comment:
You may also want to link to the connectv2 transaction model and note that this value corresponds to the `client_id` field in that model: https://docs.connect.squareup.com/api/connect/v2/#type-transaction